src/oscar/apps/order/abstract_models.py

```diff
@@ -134,6 +134,13 @@ def set_status(self, new_status):
     set_status.alters_data = True
 
     @property
+    def status_verbose_name(self):
+        status_verbose_names = getattr(settings, 'OSCAR_LINE_STATUS_VERBOSE_NAMES', {})
```

Comment:
Intended to allow different settings for orders and lines. What would you suggest, one settings for both?